Vagrantfile

```diff
@@ -22,7 +23,7 @@ Vagrant.configure("2") do |config|
     vb.customize ["modifyvm", :id, "--cpus", 2]
     vb.customize ["modifyvm", :id, "--ioapic", "on"]
     vb.name = "geowave-vagrant"
-    vb.gui = true
+#    vb.gui = true
```

Comment:
I thought it would be useful, but if nobody wants it we can comment it out.
On Jun 16, 2015 5:36 PM, "rfecher" <notifications@github.com> wrote:

> In Vagrantfile
> <https://github.com/ngageoint/geowave-vagrant/pull/3#discussion_r32573870>
> :
>
> > @@ -22,7 +23,7 @@ Vagrant.configure("2") do |config|
> >      vb.customize ["modifyvm", :id, "--cpus", 2]
> >      vb.customize ["modifyvm", :id, "--ioapic", "on"]
> >      vb.name = "geowave-vagrant"
> > -    vb.gui = true
> > +#    vb.gui = true
>
> @jwomeara <https://github.com/jwomeara> were you using the gui?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/ngageoint/geowave-vagrant/pull/3/files#r32573870>.
>